src/Card.jsx

```diff
@@ -1,12 +1,29 @@
 import * as React from 'react'
 import './Card.css'
 
-export default ({ isActive }) => {
+export default ({
+  isBookmarkActive,
+  title,
+  question,
+  answer,
+  isAnswerVisible,
+  tags,
+}) => {
   return (
     <section className="CardPart">
-      <div className={isActive ? 'CardBookmarkSelected' : 'CardBookmark'}></div>
-      <h2>Title</h2>
-      <p>Lorem ipsum dolor sit amet.</p>
+      <div
+        className={isBookmarkActive ? 'CardBookmarkSelected' : 'CardBookmark'}
+      ></div>
+      <h2>{title}</h2>
+      <p>{'Frage: ' + question}</p>
+      <p className={isAnswerVisible ? '' : 'InvisibleAnswer'}>
+        {'Antwort: ' + answer}
+      </p>
+      <ul className="TagList">
+        {tags.map(tag => (
+          <li className="TagList-Tag">{tag}</li>
```

Comment:
React needs a `key` here. 
```suggestion
          <li key={tag} className="TagList-Tag">{tag}</li>
```